### ISSUE TITLE
Government requested misinfo policy?

### ISSUE THREAD

**Issue by tomaszpieczykolan** [twitter/the-algorithm#502]
https://github.com/twitter/the-algorithm/blob/ec83d01dcaebf369444d75ed04b3625a0a645eb9/visibilitylib/src/main/scala/com/twitter/visibility/models/MisinformationPolicy.scala#L174

I'm browsing through some random files and I stumbled upon this interesting line...

**Comment by edmundpf** [comment]
Send this to Matt Taibbi

**Comment by tomaszpieczykolan** [comment]
There are also other even more interesting things here: https://github.com/twitter/the-algorithm/blob/ec83d01dcaebf369444d75ed04b3625a0a645eb9/visibilitylib/src/main/scala/com/twitter/visibility/models/TweetSafetyLabel.scala#L175

but it seems like it is already deprecated, if I'm understanding the code correctly 😅

**Comment by edmundpf** [comment]
> There are also other even more interesting things here:
> 
> https://github.com/twitter/the-algorithm/blob/ec83d01dcaebf369444d75ed04b3625a0a645eb9/visibilitylib/src/main/scala/com/twitter/visibility/models/TweetSafetyLabel.scala#L175
> 
> but it seems like it is already deprecated, if I'm understanding the code correctly 😅

Just the Brazilian Election part as the election has passed.

**Comment by tomaszpieczykolan** [comment]
There's much more than just Brazilian election over there. `MsnfoFrenchElection`, `MsnfoPhilippineElection`, `MisinfoCovid19`, `PersonaNonGrata` just to name a few... All of those are deprecated, but I'm curious about why they were used in the past... 🤔

**Comment by ValZapod** [comment]
>about why they were used in the past...

Guess.

**Comment by darkdevildeath** [comment]
I'm Brazilian and I explain. In Brazil there are laws that companies must follow about disinformation and hate speech.
Unlike the USA, in Brazil freedom of expression has clear limits where your freedom of expression cannot violate other rights, such as life.
Here in Brazil, a company cannot use freedom of expression to make false advertising. And the press cannot use free speech to say that someone is a criminal before the accused is convicted in court.

That said, Brazilian superior courts often meet with large technology companies to work cooperatively on techniques to combat crimes against Brazilian law. These meetings are public and transparent. Any citizen can access them.
No company is obliged to follow an agreement. But by not doing so, if a crime occurs within a social network controlled by that company, justice may be less flexible. Claiming that the company generally does not cooperate with justice.
These collaborations may include sharing information about terrorist factions (such as the PCC), kidnappings, murders, rapes and child sex trafficking. Also include crimes of opinion. For example, if I say that black people should die, this is a serious crime in Brazil and companies collaborate to help punish those who commit it.

**Comment by ValZapod** [comment]
>Unlike the USA, in Brazil freedom of expression has clear limits where your freedom of expression cannot violate other rights, such as life.

That is false. In fact in USA threating a president's life is a big federal crime. And in Brazil I would imagine threating lives by the polticians and public people is allowed, just like in USA. First have aboslute immunity, except for treason and seditious conspiracy, and presidents only high crimes; second have protections because you need not just threaten a life but have actual malice.

**Comment by darkdevildeath** [comment]
@ValZapod
> > Unlike the USA, in Brazil freedom of expression has clear limits where your freedom of expression cannot violate other rights, such as life.
> 
> That is false. In fact in USA threating a president's life is a big federal crime. And in Brazil I would imagine threating lives by the polticians and public people is allowed, just like in USA. First have abosolute immunity, except for treason and seditious conspiracy, and presidents only high crimes and not enough political power; second have protections because you need not just threaten a life but have actual malice, like lie so that they will get executed.

In Brazil, only parliamentarians have speech immunity. This immunity has limits. If a parliamentarian says "blacks should die" they crosses the limits of immunity and is condemned for it. A recent example is the case of a congressman who said that citizens of a northeastern state should not be employed in the south. The simple fact that he said this in a context where people from the northeastern state were being enslaved by companies from the south revealed in a police operation was enough to accuse the parliamentarian of a crime exceeding the immunity limit.

The same is true of the judiciary. If a judge violates opinion limits, they suffers punishment. But they are milder and involve loss of office and a fine. Except in the case of racism or homophobia. Here this crime has punishments as severe as murder.
Judges have additional limits, such as a ban on defending political agents on social media.

The president of the republic does not have speech immunity. But they is privileged that they can only be tried by the supreme court while they is in office. With the end of the presidential term, they can be tried by any court for their speeches during the term.

Police and military personnel are also prohibited from carrying out political propaganda or supporting politicians on social media while they are on duty.

A classic example is a former minister who was both an active general in the army and campaigning politically. The case is on trial. In Brazil, active-duty military personnel are explicitly prohibited by law from expressing political support for anyone.

What I meant is not that restrictions don't exist in the US. And yes its more flexible, less detailed and less comprehensive.

In Brazil, you can be the Queen of England, if you call someone a monkey or disparage someone's skin color, you are treated like a murderer or rapist.

This is sometimes so intense, that if you say that you prefer pink pencils to black some people may want to attack you before you have time to explain yourself. Here tolerance for hate speech is very short.
